commons/commons-lang/src/main/java/com/clicktravel/common/random/Randoms.java

```diff
@@ -229,6 +231,107 @@ public static BigDecimal randomPositiveBigDecimal(final int maxValue, final int
     }
 
     /**
+     * Return an int in the range <code>from</code> to <code>to</code> inclusive of from but exclusive of to e.g.
+     * randomIntInRange(1,20) could return an int from 1 to 19 inclusive. Negative numbers are also supported by this
+     * method e.g. randomIntInRange(-10,10) could return an int from -10 to 9 inclusive
+     *
+     * @param from the start of the range (inclusive)
+     * @param to the end of the range (exclusive)
+     *
+     * @return a random int
+     */
+    public static int randomIntInRange(final int from, final int to) {
+        int returnValue;
+        if (from > to) {
+            throw new IllegalArgumentException(String.format("From (%d) must be less than to (%d)", from, to));
+        } else if (from == to) {
```

Comment:
This test isn't strictly necessary, the following else clause will evaluate ```from + Random.randomInt(0)``` which always equal to ```from```